### DISCUSSION TITLE
Scaling problems: Everything looks blurry or too small

### DISCUSSION THREAD

**Discussion by ivanhernandez-dev** [prasanthrangan/hyprdots#1156] [Q&A]
I´m using a 15,6" laptop 1920x1080 res. When I installed the dotfiles everything seamed to have the correct size but when i installed some apps such as discord or intellij, they look blurry. I found out that the problem was fractional scaling so i disabled it by setting the following configuration in my hyprland.conf.

`xwayland {
  force_zero_scaling = true
}

monitor = eDP-1,1920x1080@144,0x0,1

env = GDK_SCALE,1`

But now everything is so tiny to the point where you can't even read it. ¿Is there any way to solve this (making everything bigger without blur)?

Thanks in advance. Please ask me any question needed if I haven provided enough information (sorry I'm newbie in this).

**Comment by kRHYME7** [comment]
``` env = GDK_SCALE,2  ```

scale it to 2 but it would be bigger.  I willl suggest running those apps (if ssupported ) in wayland. Search on what flags or settings need changing to fix it. 

Some apps can just be zoomed (like Discord) so run it on wayland then zoom just like a browser ``` ctrl + = ``` or ``` ctrl + - ```

**Reply by ivanhernandez-dev** [reply]
I had already tried that option but x2 is too big. Moreover I would like to not have to manually configure the zoom/size of each app.

**Reply by kRHYME7** [reply]
Sadly it's either the app supports wayland or not. Also if users can tolerate the blurring of non fractional scaling  due to the app running on xwayland. Have you tried passing wayland flags for the apps?

**Reply by ivanhernandez-dev** [reply]
How do I do that?

**Reply by kRHYME7** [reply]
Can you list down all the apps that re blurred?

**Reply by ivanhernandez-dev** [reply]
Sure.

- phpstorm
- intellij idea ultimate
(these two were installed from jetbrains toolbox)
- discord
- visual studio code (unless i run it with command vc from terminal but looks blurry from rofi)
- brave

**Reply by kRHYME7** [reply]
Hey, I added some wiki discussions on how to fix this blurry apps.
 Please try to follow it first 
https://github.com/prasanthrangan/hyprdots/issues/1168#issuecomment-2016754366
It should probably fix most of the issue.

Please ask for clarifications, and if those fixes don't work, don't hesitate to ask.

**Comment by rubiin** [comment]
similar issue. i am using something like this but rofi seems to be too big. I just played around with the values and this is the most sensible look
```
env = GDK_SCALE,1

xwayland {
    force_zero_scaling = true
    }

monitor=,highres,auto,1.3333

```
